client/next.config.js

```diff
@@ -2,6 +2,7 @@
 const nextConfig = {
   reactStrictMode: true,
   swcMinify: true,
+  output: 'standalone',
   publicRuntimeConfig: {
     API_URL: "http://localhost:8080",
```

Comment:
thanks for moving things to next.js environment variables! can you update the `API_URL` nested in the `publicRuntimeConfig` to use them as well? 

```API_URL: "http://" + process.env.NEXT_PUBLIC_API_HOSTNAME || "http://localhost:8080",```

Reply:
Arranged the runtime config - now we have 2 base URLs - for API and for Board (websockets)